### DISCUSSION TITLE
Properties of undefined and not sure why

### DISCUSSION THREAD

**Discussion by gdlninja** [artilleryio/artillery#1213] [Artillery]
Am running a simple test calling my api (get) passing in a requestid to get databack.

Receive the following error and not sure why.
worker error, id: 4 TypeError [Error]: Cannot read properties of undefined (reading 'undefined')
    at Object.picker (/usr/local/lib/node_modules/artillery/core/lib/weighted-pick.js:22:20)
    at runScenario (/usr/local/lib/node_modules/artillery/core/lib/runner.js:286:20)
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/artillery/core/lib/runner.js:174:25)
    at EventEmitter.emit (/usr/local/lib/node_modules/artillery/node_modules/eventemitter3/index.js:181:35)
    at UniformProcess.<anonymous> (/usr/local/lib/node_modules/artillery/core/lib/phases.js:194:10)
    at UniformProcess.emit (node:events:390:28)

HTTP request, not socketio
phases 
phases:
      - duration: 60
        arrivalRate: 5
        name: Warm up
      - duration: 120
        arrivalRate: 5
        rampTo: 50
        name: Ramp up load
      - duration: 600
        arrivalRate: 50
        name: Sustained load

**Comment by hassy** [comment]
Looks like this error is output when the test script has no scenarios - can you confirm that the script you're running has both a `config` and a `scenarios` section?

https://artillery.io/docs/guides/guides/test-script-reference.html#Overview

**Reply by gdlninja** [reply]
Taking out some information but this is my yaml file.

config:
  target: "https://myurl"
  phases:
      - duration: 10
        arrivalRate: 1
        name: Sustained load
  defaults:
    headers:
      Content-Type: "application/json"
      X-API-Key: "my key"    
scenarios:
    - flow:
      - get:
          url: "/api/v1/ibv/requests/requestid"

**Reply by hassy** [reply]
works here with the latest release of Artillery... what version of Artillery are you running on? (`artillery -V`)

**Reply by ChristianRich** [reply]
I found, that when copy / pasting the sample scripts from the website, the tabbing is broken and you get strange errors. So be mindful.

**Reply by hassy** [reply]
oh that's not good, thanks for letting us know @ChristianRich!

**Comment by gdlninja** [comment]
looks like it was tabbing issue.
